codespell_lib/data/dictionary.txt

```diff
@@ -17661,6 +17806,8 @@ extered->exerted
 extereme->extreme
 extermal->external, extremal,
 exterme->extreme
+extermely->extremely
```

Comment:
Perhaps add comparative?
```
extermer->extremer
```

Reply:
Done. I also added `extermity->extremity` and its plural.